src/components/Calculator.js

```diff
@@ -1,36 +1,55 @@
+/* eslint-disable class-methods-use-this */
 import React from 'react';
+import calculate from '../logic/calculate';
+// import calculate from '../logic/calculate';
```

Comment:
- [ ] [OPTIONAL] It's not a hard requirement right now, but always delete your commented code, it's a good practice

Reply:
Thank you!, I forgot about that one good catch!